tools/verify-commit/verify-commit.go

```diff
@@ -38,6 +38,10 @@ const dependabot = "dependabot[bot]"
 
 func validate(commit GitCommit) error {
 	var errs error
+	if commit.Author == "red-hat-konflux[bot]" {
```

Comment:
@shajmakh  
1. maybe we should do this in a dedicated PR so we can track it properly ?
2. The is was DCO mismatched, should we keep this addition as it is?

Reply:
Thanks for your comment. 
1. sure, sounds better
2. I'm not sure I get the question, I think for now we only need to relax the verification on konflux's bot commits. I actually went through few k5x bot PRs and it has these 2 versions:
```
Signed-off-by: red-hat-konflux <126015336+red-hat-konflux[bot]@users.noreply.github.com>
```
and 
```
Signed-off-by: red-hat-konflux <konflux@no-reply.konflux-ci.dev>
````
I'll update the if statement to include both versions and file a new PR.

Reply:
filed:https://github.com/openshift-kni/numaresources-operator/pull/1298 please let me know if this is addressing your question. Thanks.